pkg/hub/controllers/vpnkeyrotation/reconciler.go

```diff
@@ -112,7 +112,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (ctrl
 				if err := r.updateRotationStatus(ctx, selectedGw, hubv1alpha1.SecretUpdated, vpnKeyRotation); err != nil {
 					return ctrl.Result{}, err
 				}
-				return ctrl.Result{Requeue: true}, nil
+				// return ctrl.Result{Requeue: true}, nil
```

Comment:
if you update and don't requeue; it will error out to conflict error in the future; hence I requeued here.

Reply:
No It is not going and this requeue was unnecessary and breaking the UTs too. So I removed

Reply:
@YachikaRalhan please remove the commented code